### PR TITLE
Test, Debug, Troubleshoot updates

### LINKED ISSUE #6689: Enhance the Test, debug, and troubleshoot index page
[Test, debug, and troubleshoot](https://docs.microsoft.com/aspnet/core/test)

Two enhancements will promote dev success:

1. ~We have additional troubleshooting guidance around the repo that should be cross-linked into this index page.~ Found it in the *Troubleshooting ASP.NET Core projects* topic. We're good on this one.
1. There's no indication on what the linked content addresses. Each should have a short, 1-3 sentence description.

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #6689 

* Polish up the 🍎 on this landing page.
* Change the error messages to block quotes in the *Troubleshooting ASP.NET Core projects* topic and fix the title.

### CODE REVIEW

File: aspnetcore/test/troubleshoot.md [hunk 1]
```diff
@@ -19,56 +19,59 @@ The following links provide troubleshooting guidance:
...
 
- The .NET Core SDK is installed in multiple locations. Only templates from the SDK(s) installed at 'C:\Program Files\dotnet\sdk\' will be displayed.
+In the **New Project** dialog for ASP.NET Core, you may see the following warning:
+
+> The .NET Core SDK is installed in multiple locations. Only templates from the SDK(s) installed at 'C:\\Program Files\\dotnet\\sdk\\' will be displayed.
 
 ![A screenshot of the OneASP.NET dialog showing the warning message](troubleshoot/_static/multiplelocations.png)
 
-You see this message when you have at least one installation of the .NET Core SDK in a directory outside of *C:\Program Files\dotnet\sdk\*. Usually that happens when the .NET Core SDK has been deployed on a machine using copy/paste instead of the MSI installer.
+You see this message when you have at least one installation of the .NET Core SDK in a directory outside of *C:\\Program Files\\dotnet\\sdk\\*. Usually this happens when the .NET Core SDK has been deployed on a machine using copy/paste instead of the MSI installer.
```
Comment (Rick-Anderson) [inline aspnetcore/test/troubleshoot.md]: why not use forward slash? Folks know what you mean. `c:/Program Files/dotnet/sdk` works when you copy/paste and looks alot better in raw format.

@tdykstra  what do you think?

Comment (guardrex) [reply aspnetcore/test/troubleshoot.md]: I agree, but it was a Windows directive I think. I think the guidelines are when only talking about Windows to use the backslash.

Comment (guardrex) [reply aspnetcore/test/troubleshoot.md]: Yes, the guideline is ...

> To indicate a path, first type the drive name, followed by a colon and a backslash. Then type the name of each folder in the order that you would open it, separated by a backslash, as follows:
>
> C:\Documents and Settings\user1

Comment (guardrex) [reply aspnetcore/test/troubleshoot.md]: Do you want me to go ahead anyway with forward slashes?

Comment (tdykstra) [reply aspnetcore/test/troubleshoot.md]: Where did that quotation come from?

Comment (Rick-Anderson) [reply aspnetcore/test/troubleshoot.md]: @guardrex  keep what you have, but I'd like to think about using my approach.

Comment (guardrex) [reply aspnetcore/test/troubleshoot.md]: MS Writing Style Guide - Section 5.18.8 path

**Pull request merged**